### ISSUE TITLE
在MVSA-Multiple以及HFM数据集上，训练模型时loss出现负值

### ISSUE THREAD

**Issue by DavidSmith-FHTT** [SorF520/D2R#2]
您好，在MVSA-Multiple以及HFM数据集上，训练模型时 loss 出现负值，请问这是正常现象吗？

MVSA-Multiple： 04/21/2025 23:52:23 - INFO - __main__ -     **loss = -2.2620671689510345**
HFM：04/22/2025 00:05:37 - INFO - __main__ -     **loss = -24.417478680610657**
